lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -126,6 +126,12 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
         }
     }
 
+    ngAfterViewInit() {
```

Comment:
all this change afterViewInit should be avoided

Reply:
I have removed that part